typo3conf/LocalConfiguration.php

```diff
@@ -9,7 +9,7 @@
     ],
     'DB' => [
         'database' => 't3kit',
-        'host' => 'localhost',
+        'host' => 'db',
```

Comment:
I am sorry to late reply @laurisaarni, @MattiasNilsson I try it already, it does not work for docker. It should be `db`